packages/logo/lib/src/logo/components/logo/logo.component.html

```diff
@@ -1,4 +1,3 @@
-<a class="o-header__logo" href="{{ link || '#' }}" title="{{ title }}" (click)="onClick($event)">
+<a href="#" class="o-header__logo" (click)="logoClicked($event)">
```

Comment:
is it not better to leave `link  in the href so that "open in new tab" will work?